monkey/infection_monkey/post_breach/post_breach_handler.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import infection_monkey.config
+import platform
+from file_execution import FileExecution
+from pba import PBA
+
+LOG = logging.getLogger(__name__)
+
+__author__ = 'VakarisZ'
+
+
+# Class that handles post breach action execution
+class PostBreach(object):
+    def __init__(self):
+        self.os_is_linux = False if platform.system() == 'Windows' else True
+        self.pba_list = self.config_to_pba_list(infection_monkey.config.WormConfiguration)
+
+    def execute(self):
+        for pba in self.pba_list:
+            pba.run(self.os_is_linux)
+        LOG.info("Post breach actions executed")
+
+    @staticmethod
+    def config_to_pba_list(config):
+        """
+        Returns a list of PBA objects generated from config.
+        """
+        pba_list = []
+        pba_list.extend(PostBreach.get_custom(config))
+
+        return pba_list
+
+    @staticmethod
+    def get_custom(config):
```

Comment:
this and a lot of other PBA functions returns lists of PBAs. but they're always of size 1 (sometimes 0),
Is there any justification for the whole list thing? It just complicates everything all along the code

Reply:
This method returns 0, 1, or 2 objects.